libs/config/cmake/templates/config_version.hpp.in

```diff
@@ -70,8 +70,7 @@ namespace hpx
 }
 
 ///////////////////////////////////////////////////////////////////////////////
-#if !defined(HPX_EXPORTS) && !defined(HPX_MODULE_EXPORTS) &&                   \
-    !defined(HPX_NO_VERSION_CHECK)
+#if !defined(HPX_EXPORTS) && !defined(HPX_NO_VERSION_CHECK)
```

Comment:
Are we sure this can be removed? IIRC, `HPX_MODULE_EXPORTS` is defined by other shared libraries (non-core HPX) to avoid generating the version checks.

Reply:
My impression was that `HPX_MODULE_EXPORTS` was defined for the static module libraries, but `HPX_EXPORTS` works just as well. I do have to verify this still though. Parcelports also defined it but there again `HPX_EXPORTS` works just as well since they're built into `libhpx`. If they are to work as standalone plugins again they might need another definition.

Reply:
This works fine.